src/dotty/tools/dotc/sbt/ShowAPI.scala

```diff
@@ -0,0 +1,156 @@
+// This file is copied straight from
```

Comment:
When running dotty outside of sbt, we only have `"org.scala-sbt" % interface` (which only contains Java interfaces) on the classpath, `ShowAPI.scala` is in `"org.scala-sbt" % api` which will only be on the classpath when running dotty from sbt, but we want `-Ydump-sbt-inc` to work without running dotty from sbt to make debugging easier.